source/simulate/replicate-simulation-inputs.md

```diff
@@ -163,9 +165,9 @@ Note the difference in nesting: we no longer have an inner set of two axes, but
 64
 ```
 
-Why? Because we are varying `age` and `salary` independently, and each axis results in multiplying by 4 our original population of 4, resulting in 4 times 4 times 4 individuals, which is indeed 64.
+Why? Because `age` and `salary` are varying independently and each axis results in multiplying by 4 the original population of 4. This resultS in 4 times 4 times 4 individuals, equalling 64.
```

Comment:
Typo:
```suggestion
Why? Because `age` and `salary` are varying independently and each axis results in multiplying by 4 the original population of 4. This results in 4 times 4 times 4 individuals, equalling 64.
```